src/flick/friend/views.py

```diff
@@ -133,3 +136,15 @@ def post(self, request, format=None):
         serializer = FriendUserSerializer(friends_removed, many=True)
 
         return success_response(serializer.data)
+
+
+class FriendUserView(generics.GenericAPIView):
+    model = Profile
+    serializer_class = FriendProfileSerializer
+    permission_classes = api_settings.CONSUMER_PERMISSIONS
+
+    def get(self, request, pk):
+        if not User.objects.filter(id=pk):
+            return failure_response(f"User of id {pk} not found.")
+        profile = Profile.objects.get(user_id=pk)
```

Comment:
hm this shouldn't run at all -- I'm surprised tests pass? should be `Profile.objects.get(user__id=pk)`